python/doc/examples/functional_modeling/functional_basis/plot_multidimensional_basis.py

```diff
@@ -0,0 +1,59 @@
+"""
+============================================================================
+Create a basis of functions with input and output dimension greater than one
+============================================================================
+"""
+# %%
+# Import the required modules
+
+import openturns as ot
+
+# %%
+# Description
+# ===========
+#
+# We want to build a basis :math:`\mathcal{B}=\{f_1,...,f_n\}` such that :math:`f_n: \mathbb{R}^d \mapsto \mathbb{R}^q` with :math:`d>1, q>1`.
```

Comment:
Let my clarify what I mean by making a link to the `AggregatedFunction` class.
```suggestion
# We want to build a basis :math:`\mathcal{B}=\{f_1,...,f_n\}` such that :math:`f_n: \mathbb{R}^d \mapsto \mathbb{R}^q` with :math:`d>1, q>1`.
# In other words, we want to create a basis of functions having both a multivariate input *and* output.
# In this example, he present methods to use the :class:`~openturns.AggregatedFunction` class to
# create a multivariate output function from a list of functions having a univariate output dimension.
```

Reply:
ok I uncluded the text.